spec/ics-002-client-semantics/README.md

```diff
@@ -256,6 +256,27 @@ Once misbehaviour is detected, clients SHOULD be frozen so that no future update
 A permissioned entity such as a chain governance system or trusted multi-signature MAY be allowed
 to intervene to unfreeze a frozen client & provide a new correct header.
 
+#### Height
+
+`Height` is an opaque data structure defined by a client type.
+It must form a partially ordered set & provide operations for comparison.
+
+```typescript
+type Height
+```
+
+```typescript
+enum Ord {
```

Comment:
can we rename it to  `Cmp` instead? it's more widely used by various libraries in Go (eg big.Int)

Reply:
actually, I think Cmp should be a function from `Height`

```go
func (h Height) Cmp(b Height): int {
  if h.epochNumber < b.epochNumber {
    return -1
  } else if h.epochNumber == b.epochNumber {
    if a.epochHeight < b.epochHeight {
      return -1
    } else if h.epochHeight == b.epochHeight {
      return 0
    }
  }
  return 1
}
```

something in those lines

Reply:
https://golang.org/pkg/math/big/#Int.Cmp

Reply:
Hmm, I'm not a huge fan of returning an `int` since there are more valid values for `int` than valid return values for this function. It's fine if we use `Cmp` to implement this in the Cosmos SDK code, though.